aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -197,6 +198,9 @@ static size_t _getWorkspaceSize() {
   return workspace_size;
 }
 
+inline bool isFloat8Type(::c10::ScalarType t) {
```

Comment:
Should we have this with the other dtype utils in c10::ScalarType as I'm sure we'll need this in other places?

Reply:
Good point, refactored into https://github.com/pytorch/pytorch/pull/106977